tests/unit/components/choropleth.spec.ts

```diff
@@ -7,12 +7,7 @@ const store = mockPinia();
 const renderComponent = () => {
     return render(Choropleth, {
         global: {
-            plugins: [store],
-            stubs: {
-                LMap: true,
-                LTileLayer: true,
-                LGeoJson: true
-            }
```

Comment:
I removed these because they didn't seem to affect whether the rendered Choropleth component contained <l-map-stub> or not (these leaflet components are stubbed in testSetup.ts). If they are supposed to achieve something else besides, then let me know.

Reply:
No that's fine, I guess this was superseded by the setup code.